### PR TITLE
personal profile

### PR DESCRIPTION
**Pull request by namnn17b3**
## Related Tickets
https://edu-redmine.sun-asterisk.vn/issues/74041

## WHAT (optional)
- Change number items `completed/total` in admin page.

## HOW
- I edit js file, inject not_vary_normal items in calculate function.

## WHY (optional)
- Because in previous version - number just depends on `normal` items. But in new version, we have `state` and `confirm_state` depends on both `normal` + `not_normal` items.

## Evidence (Screenshot or Video)
image
image
image
image
image
image
image
image
image
image
image

### CODE REVIEW

**Comment by namnn17b3** [comment]
ready

File: src/apis/user.api.controller.ts [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+import { UserUpdateRequestDto } from '../dtos/req/user/user.update.req.dto';
+import { PipeDto } from '../decoratos/api/pipe.decorator';
+import session from 'express-session';
+import { ChangePasswordDto } from '../dtos/req/user/change.password.dto';
+import { Bcrypt } from '../security/bcrypt';
+import { ErrorApiResponseDto } from '../dtos/res/error.api.res.dto';
+import { ImageFileGuard } from '../decoratos/api/guards/guard.image.file.decorator';
+
+@RestController('/api/user')
+export class UserRestContrller {
```
Comment (huongnt-2545) [inline src/apis/user.api.controller.ts]:
```suggestion
export class UserRestController {
```

File: src/public/js/personal-info.js [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+  'oldPassword': 'Mật khẩu cũ không khớp',
+  'newPasword': 'Mật khẩu mới có độ dài từ 8 đến 24 kí tự',
+  'confirmPassword': 'Xác nhận mật khẩu mới không khớp'
+}
+
+const translateEnglish = {
+  'oldPassword': 'Old Password not match',
+  'newPasword': 'New Password length must be from 8 to 24 characters',
+  'confirmPassword': 'Confirm Password not match'
+}
```
Comment (huongnt-2545) [inline src/public/js/personal-info.js]: thêm value translate tương ứng với line 23 nhé

**Comment by namnn17b3** [comment]
ready

**Review by huongnt-2545 — APPROVED** [review]
merged

**Pull request merged**